src/components/expression/expression-renderer.js

```diff
@@ -465,22 +604,51 @@ genotet.ExpressionRenderer.prototype.drawGeneProfiles_ = function() {
       this.PROFILE_MARGINS_.LEFT = this.DEFAULT_PROFILE_MARGIN;
     }
     else {
-      this.PROFILE_MARGINS_.LEFT = this.geneLabelWidth_ + this.LABEL_MARGIN_;
+      this.PROFILE_MARGINS_.LEFT = this.geneLabelWidth_ + this.LABEL_MARGIN;
```

Comment:
You are changing a constant variable, which isn't allowed though we are not blocking it in compilation.